Bottom: 9b280edd294961ed861f2ee0b3eb68d5143312a4
Top:    1f4a24e77a80d749c73db759d2e97215c4d530ee
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 16:31:27 +0000

Subtract vias from ...


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 71dfc12..08ef9ef 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1267,6 +1267,10 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
    *
    */
 
+    fprintf (stderr, "Subtracting pin + via drills\n");
+    r_search (PCB->Data->pin_tree, &bounds, NULL, pv_drill_callback, &info);
+    r_search (PCB->Data->via_tree, &bounds, NULL, pv_drill_callback, &info);
+
   if (info.poly == NULL)
     {
       fprintf (stderr, "Skipping layer group %i, info.poly was NULL\n", group);
